### DISCUSSION TITLE
EADesktop and Ubisoft Connect should be bot mantained

### DISCUSSION THREAD

**Discussion by auanasgheps** [microsoft/winget-pkgs#119110] [General]
ElectronicArts.EADesktop and Ubisoft.Connect update too much frequently to be manually maintained. 
Yesterday I created two PRs and are already out of date, because a new release has been pushed for each of these packages.

The download link for these packages is static, why there aren't bots that are performing these updates?

**Comment by mdanish-kh** [comment]
> The download link for these packages is static, why there aren't bots that are performing these updates?

- https://github.com/microsoft/winget-pkgs/issues/111576

These packages used to be automated by wingetbot before it stopped functioning.

- https://github.com/microsoft/winget-pkgs/pulls?q=is%3Apr+ElectronicArts.EADesktop+author%3Awingetbot+is%3Aclosed+
- https://github.com/microsoft/winget-pkgs/pulls?q=is%3Apr+Ubisoft.Connect+author%3Awingetbot+is%3Aclosed+

**Reply by auanasgheps** [reply]
Thanks for letting me know! So the bot is down and is up to Microsoft to fix it.
